x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_form.tsx

```diff
@@ -311,6 +331,14 @@ export const StepDefineForm: FC<StepDefineFormProps> = React.memo((props) => {
                     </EuiFlexItem>
                   </EuiFlexGroup>
                 )}
+                <FullTimeRangeSelector
+                  frozenDataPreference={frozenDataPreference}
+                  setFrozenDataPreference={setFrozenDataPreference}
```

Comment:
A change to the frozen data preference is currently acting as a submit on the 'Use full data' button. I don't think this should trigger an event here - added an item to the parent issue https://github.com/elastic/kibana/issues/146187.

Reply:
PR with a fix for this is up here: #151673